internal/server/statvar_group.go

```diff
@@ -235,32 +235,30 @@ func (s *Server) GetStatVarGroupNode(
 			return nil, err
 		}
 		// Count for current node.
-		if existence, ok := statVarCount[svg]; ok && len(existence) == len(places) {
+		result.NumDescendentStatVars = 0
+		if existence, ok := statVarCount[svg]; ok && len(existence) > 0 {
 			for _, count := range existence {
-				// Use the smallest count among all places.
-				if count < result.NumDescendentStatVars {
+				// Use the largest count among all places.
+				if count > result.NumDescendentStatVars {
```

Comment:
Is this still an under estimation, if the SVs for the places are non-overlapping?  But probably okay for now...